src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -120,6 +120,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@SteveL-MSFT, your last commit had 11 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

File encoding tests.File contents are UTF8 without BOM.Appends correctly on non-Windows systems
```powershell
Expected strings to be the same, but they were different.
Expected length: 45
Actual length:   22
Strings differ at index 22.
Expected: '...16-10-116-...'
But was:  '116-195-169-115-116-10'
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/engine/Basic/Encoding.Tests.ps1: line 74
74:             $bytes -join "-" | Should -Be ($Expected -join "-")
```
ConsoleHost unit tests.CustomPipeName startup tests.Should create pipe file if CustomPipeName is specified
```powershell
Expected $true, but got $null.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 757
757:             & $powershell -CustomPipeName $pipeName -Command "Test-Path '$pipePath'" | Should -BeTrue
```
Invoke-Item basic tests.Should invoke an executable file without error
```powershell
Expected regular expression 'usage: ping|ping:' to match '&27;[91m/Users/runner/runners/2.166.4/work/1/s/Invoke-Item: &27;[91mThe term '/Users/runner/runners/2.166.4/work/1/s/Invoke-Item' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.&27;[0m
', but it did not match.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1: line 155
155:             Get-Content $redirectFile -Raw | Should -Match "usage: ping|ping:"
```
MSFT:3309783.Run in another process
```powershell
Expected 'System.Management.Automation.PSObject', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Language/Parser/LanguageAndParser.TestFollowup.Tests.ps1: line 44
44:         & $powershellexe -noprofile -command "[psobject] | ForEach-Object FullName" | Should -Be System.Management.Automation.PSObject
```
Native Command Arguments.Should handle spaces between escaped quotes
```powershell
Expected 2, but got 3.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Language/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1: line 32
32:         $lines.Count | Should -Be 2
```